packages/bpk-component-carousel/README.md

```diff
@@ -0,0 +1,31 @@
+# bpk-component-carousel
+
+> Backpack example component.
+
+## Installation
+
+```sh
+npm install bpk-component-carousel --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkCarousel from 'bpk-component-carousel';
+
+export default () => (
+  <BpkCarousel>
+    <BpkCard>Item 1</BpkCard>
+    <BpkCard>Item 2</BpkCard>
+  </BpkCarousel>
+);
```

Comment:
We should add a bit more guidance here to let consumers know that all items should be the same width.